package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "dok-chain-ui",
+  "version": "0.0.0",
+  "license": "MIT",
```

Comment:
Can this be made Apache 2.0?

Reply:
Absolutely, will update this and make the requested licensing additions.